File: AIPSprojekat/2OrderLibrary/Dostava.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace _2OrderLibrary
{
    public class Dostava
    {
        public int Id { get; set; }
        //public int KorisnikId { get; set; }
        public Korisnik Korisnik { get; set; }

        public int StavkaMenijaId { get; set; }
        public Meni StavkaMenija { get; set; }

        public string Sifra { get; set; }
        public DateTime Vreme { get; set; }
        public string Adresa { get; set; }
    }
}

File: 2order-frontend/src/model/Dostava.ts
import Korisnik from "./Korisnik";
import Meni from "./Meni";
import Sto from "./Sto";

interface Dostava {
    id: number;
    korisnikId: number;
    korisnik: Korisnik;
    stavkaMenijaId: number;
    stavkaMenija: Meni;
    sifra: string;
    vreme: Date;
    adresa: string;
}

export default Dostava;